src/Microsoft.ML.Core/Prediction/TrainerInfo.cs

```diff
@@ -36,12 +36,19 @@ public sealed class TrainerInfo
         public bool WantCaching { get; }
 
         /// <summary>
-        /// Whether the trainer supports validation sets via <see cref="TrainContext.ValidationSet"/>. Not implementing
-        /// this interface and returning <c>true</c> from this property is an indication the trainer does not support
+        /// Whether the trainer supports validation set via <see cref="TrainContext.ValidationSet"/>. Not implementing
+        /// this interface and returning <c>false</c> from this property is an indication the trainer does not support
         /// that.
         /// </summary>
         public bool SupportsValidation { get; }
 
+        /// <summary>
+        /// Whether the trainer can use test set via <see cref="TrainContext.TestSet"/>. Not implementing
+        /// this interface and returning <c>false</c> from this property is an indication the trainer does not support
+        /// that.
+        /// </summary>
+        public bool SupportsTest { get; }
```

Comment:
>SupportsTest [](start = 20, length = 12)

Why do you need this?

Reply:
It's used in command line family (for example, `TrainCommand.cs`). If a trainer says it can't support test set, the command used may to produce a `RoleMappedData` for the specified test set.